java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -3894,6 +3895,27 @@ public void type_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void rename() {
+        // setup
+        String key = "key1";
+        String newKey = "key2";
+        String[] arguments = new String[] {key, newKey};
+        CompletableFuture<String> testResponse = new CompletableFuture<>();
+        testResponse.complete("OK");
```

Comment:
I think there is an imported constant for this already
```suggestion
        testResponse.complete(OK);
```